privilege/privileges/privileges_test.go

```diff
@@ -732,6 +734,30 @@ func (s *testPrivilegeSuite) TestDefaultRoles(c *C) {
 	c.Assert(len(ret), Equals, 0)
 }
 
+func (s *testPrivilegeSuite) TestUserTableConsistency(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("create user superadmin")
+	tk.MustExec("grant all privileges on *.* to 'superadmin'")
```

Comment:
in order to cover all privilege, here should be `grant all privileges on *.* to 'superadmin' with grant option`